Gems/EMotionFX/Code/EMotionFX/Exporters/ExporterLib/Exporter/SkeletalMotionExport.cpp

```diff
@@ -56,12 +56,57 @@ namespace ExporterLib
         }
     }
 
+    void SaveRootMotionExtractionData(MCore::Stream* file, EMotionFX::Motion* motion, MCore::Endian::EEndianType targetEndianType)
+    {
+        if (!motion->GetRootMotionExtractionData())
+        {
+            return;
+        }
+
+        AZ::SerializeContext* serializeContext = nullptr;
+        AZ::ComponentApplicationBus::BroadcastResult(serializeContext, &AZ::ComponentApplicationBus::Events::GetSerializeContext);
+        if (!serializeContext)
+        {
+            AZ_Error("EMotionFX", false, "Can't get serialize context from component application.");
+            return;
+        }
+
+        AZStd::vector<AZ::u8> buffer;
+        AZ::IO::ByteContainerStream<AZStd::vector<AZ::u8>> stream(&buffer);
+        const bool result = AZ::Utils::SaveObjectToStream<EMotionFX::RootMotionExtractionData>(
+            stream, AZ::ObjectStream::ST_BINARY, motion->GetRootMotionExtractionData().get(), serializeContext);
+        if (result)
```

Comment:
you can probably invert this if and just return to be consistent with line 70 above.